.github/workflows/unittests.yml

```
@@ -39,12 +39,9 @@ jobs:
       matrix:
         os: [ubuntu-latest]
         python:
-          - {version: '3.10'}
           - {version: '3.11'}
           - {version: '3.12'}
         include:
```

We do want to test the oldest supported py version on windows. Reason for this is things like changes in importlib behaviour.
```suggestion
        include:
          - python: {version: '3.11'}  # old compat
            os: windows-latest
```